articles/quickstarts/qrng.md

```diff
@@ -18,11 +18,33 @@ A simple example of a quantum algorithm written in Q# is a quantum random number
 - [Create a Q# Project](xref:microsoft.quantum.howto.createproject)
 
 
-## Write a Q# operation
+## Understanding the qubit
 
-As mentioned in our [What is Quantum Computing?](xref:microsoft.quantum.overview.what) article, a qubit is a unit of quantum information that can be in superposition. When measured, a qubit can only be either 0 or 1, however during execution, the state of the qubit represents the probability of reading either a 0 or a 1 during measurement. This probabilistic state is known as superposition. We can use this probability to generate random numbers.
+As mentioned in our [What is Quantum Computing?](xref:microsoft.quantum.overview.what) article, a qubit is a unit of quantum information that can be in superposition. A classical bit can only take the values **0** or **1**. A qubit can exist in a quantum superposition between the two classical values **0** and **1**. However, when we measure a qubit we can only observe the states **0** or **1**. We say that a measurement *collapses* the state the qubit from the superposition to the state corresponding the classical value that we observe.
+
+There's no unique way for a qubit to be in superposition, there are infinite ways to exist in superposition. The probabilities of obtaining either **0** or **1** when measuring the qubit depends on the specific superposition. For example, we can have a qubit in superposition in which the probability of obtaining **1** is very high, let's say a 90%. Therefore the probability of obtaining **0** is low (10%). Or maybe we have a qubit in a different superposition in which the probabilities to obtain **0** and **1** when measuring are the same (50%). As there are infinite ways for the qubit to be in a superposition, a clever way to express the state of a qubit is the *Bolch sphere*.
```

Comment:
It is common that layman people associate "superposition" with a single state and think that qubits can be in three possibilities: **0**, **1** and **both at the same time**.
But the reality is more nuanced since not all superposition yield the same probabilities of obtaining **0** and **1**. I wanted to convey this. I'll try to condense it to make it clearer.

Reply:
Agreed, @geduardo, it's really common to see readers that confuse qubits with what are effectively classical trits.